package-windows.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "freshrss",
+  "description": "A free, self-hostable aggregator",
+  "homepage": "https://freshrss.org/",
+  "readmeFilename": "README.md",
+  "bugs": {
+    "url": "https://github.com/FreshRSS/FreshRSS/issues"
+  },
+  "keywords": [
+    "news",
+    "aggregator",
+    "RSS",
+    "Atom",
+    "WebSub"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/FreshRSS/FreshRSS.git"
+  },
+  "license": "AGPL-3.0",
+  "scripts": {
```

Comment:
instead of a new file, you could have just the scripts that require a Windows-specific command, e.g. `"stylelint_win"`

Reply:
And maybe we can find a syntax that works on both platforms

Reply:
I suspect the syntax as written is already a syntax that works on both platforms. I probably just used native shell instead because it's more intuitive. Cf. https://www.npmjs.com/package/fast-glob which is used by Stylelint. (I don't have npm set up on this laptop.)

Reply:
Put another way, just change that single line in the original file, introduce an error and see if the CI catches it.

Reply:
No, the syntax without the quotes does not work in Bash, but another syntax might be possible